File: py/IEC61968/Work/InternalLocation.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106
from CIM_STD_PYTHON.TC57CIM.IEC61968.Work.WorkLocation import WorkLocation


class InternalLocation(WorkLocation):

    def __init__(self):
        super().__init__()
        self.building_name = ""
        self.building_number = ""
        self.floor = 0
        self.room_number = ""

File: java/IEC61968/Work/InternalLocation.java
package IEC61968.Work;

import IEC61970.Base.Domain.String;
import IEC61970.Base.Domain.Integer;

/**
 * @author marga
 * @version 1.0
 * @created 25-Dec-2023 8:45:22 PM
 */
public class InternalLocation extends WorkLocation {

	public String buildingName;
	public String buildingNumber;
	public Integer floor;
	public String roomNumber;

	public InternalLocation(){

	}
}//end InternalLocation